Corona-Warn-App/src/main/res/values-de/vaccination_strings.xml

```diff
@@ -87,6 +87,7 @@
         <item quantity="few">Vollständiger Impfschutz in %1$d Tagen</item>
         <item quantity="many">Vollständiger Impfschutz in %1$d Tagen</item>
     </plurals>
+    <string name="vaccination_card_status_vaccination_complete_tomorrow">Vollständiger Impfschutz ab morgen</string>
```

Comment:
We can't use `plurals.zero` because the relation between quantity types is a grammatical one, e.g. in Germany there is no difference between `other` and `zero`, so `zero` is never used.